microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -0,0 +1,242 @@
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-doc-page.msft.png" alt-text="The new documentation page that appears in the browser, about how to use Microsoft Edge Developer Tools for Visual Studio Code" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-doc-page.msft.png":::
+       The new documentation page that appears in the browser, about how to use Microsoft Edge Developer Tools for Visual Studio Code
+    :::image-end:::  
+
+1.  In the URL bar, navigate to the project that you want to debug using Microsoft Edge Developer Tools for Visual Studio Code.
+
+
+#### The Side Bar now provides a UI instead of a blank panel, to help launch your project for debugging
+
+These steps assume you are in Visual Studio Code, with no folder opened.
+
+1.  In the **Activity Bar** on the left, select the **Microsoft Edge Tools** button.  In the **Side Bar**, the **Microsoft Edge Tools: Targets** panel appears, showing a single button (**Launch Instance**):
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png" alt-text="The Microsoft Edge Tools: Targets panel in the Side Bar of Visual Studio Code, showing the Launch Instance button" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png":::
+       The **Microsoft Edge Tools: Targets** panel in the **Side Bar** of Visual Studio Code, showing the **Launch Instance** button
+    :::image-end:::  
+
+1.  In the **Microsoft Edge Tools: Targets** panel, select the **open a folder** link, and open a folder, such as your own project.  The **Microsoft Edge Tools: Targets** panel now shows two buttons: **Launch Instance** and **Generate launch.json**:
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-side-bar.msft.png" alt-text="The Microsoft Edge Tools: Targets panel in the Side Bar of Visual Studio Code" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-side-bar.msft.png":::
+       The **Microsoft Edge Tools: Targets** panel in the **Side Bar** of Visual Studio Code
+    :::image-end:::  
+    
+1.  In the **Side Bar**, select the **Generate launch.json** button.  A `launch.json` file is generated in the root of the project, and is opened for editing.
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-launch-json-file.msft.png" alt-text="Editing the URL in the launch.json file" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-launch-json-file.msft.png":::
+       Editing the URL in the `launch.json` file
+    :::image-end:::
+
+1.  In the `launch.json` file, change the URL from an empty string to a URL for your project.  Or leave the string empty, to show the documentation page.
```

Comment:
```suggestion
1.  In the `launch.json` file, change the URL from an empty string to a URL for your project.  You can leave the URL string empty to show the default documentation page described in the previous section.
```

Reply:
@codepo8 Are you recommending that I keep the existing text ("Or leave the string empty, to show the documentation page.")?  I think so, because you marked the conversation as "Resolved", which made the **Commit suggestion** and **Add suggestion to batch** buttons dimmed.  To click those buttons, I would need to first click **Unresolve conversation**.